commands/permissions.go

```diff
@@ -44,11 +53,40 @@ var ShowRoleCmd = &cobra.Command{
 	RunE:    withClient(showRoleCmdF),
 }
 
+var AssignUsersCmd = &cobra.Command{
+	Use:   "assign [role_name] [username...]",
```

Comment:
Same, and we can use `name` instead of `role_name` for consistency

```suggestion
	Use:   "assign <role> <username...>",
```